modules/ROOT/pages/hot-deployment.adoc

```diff
@@ -19,12 +19,16 @@ The application's configuration files are monitored, which means that if there a
 
 == How Hot Deployment works
 
-Mule checks every three seconds for updated configuration files under the `$MULE_HOME/apps` directory, and when it finds one, it reloads the configuration file and the JARs in that applications `java` source directory. To reload an application, you can:
+Mule checks every three seconds for updated configuration files under the `$MULE_HOME/apps` directory, and when it finds one, it reloads the configuration file and the JARs in that applications `java` source directory.
+If you start Mule by specifying an app using the `mule.deploy.applications` property, the hot deployment process works only for the specified app.
```

Comment:
If you start Mule by specifying an app using
-->
If you start Mule and specify an app using